docs-src/http-api.md

```diff
@@ -0,0 +1,530 @@
+The Ceramic HTTP API allows you to create and update documents on a remote Ceramic node. These endpoints allows you to manually make requests to a Ceramic node. This is also a good place to start if you want to build a Ceramic client in a new language. 
+
+*`ℹ️ Not all of these methods are exposed if the node runs in gateway mode.`*
+
+# Documents
+The documents endpoint is used to create new documents, load documents from their [[DocID]] or from their genesis content. 
+
+## Get the state of a document
+Load the state of a document given it's [[DocID]].
+
+### Request
+`GET /api/v0/documents/:docid`
+
+Here, `:docid` should be replaced by the string representation of the DocID of the document that is being requested.
+
+### Response
+
+The response body contains the following fields:
+
+- `docId` - the [[DocID]] of the requested document as string
+- `state` - the state of the requested document as [[DocState]]
+
+### Example
+
+```bash
+$ curl http://localhost:7007/api/v0/documents/kjzl6cwe1jw147r7878h32yazawcll6bxe5v92348cxitif6cota91qp68grbhm
+
+{
+  "docId": "kjzl6cwe1jw147r7878h32yazawcll6bxe5v92348cxitif6cota91qp68grbhm",
+  "state": {
+    "doctype": "tile",
+    "content": {
+      "Ceramic": "pottery"
+    },
+    "metadata": {
+      "schema": null,
+      "controllers": [
+        "did:key:z6MkfZ6S4NVVTEuts8o5xFzRMR8eC6Y1bngoBQNnXiCvhH8H"
+      ]
+    },
+    "signature": 2,
+    "anchorStatus": "PENDING",
+    "log": [{
+      "cid": "bagcqceramof2xi7kh6qblirzkbc7yulcjcticlcob6uvdrx3bexgks37ilva",
+      "type": 0
+    }],
+    "anchorScheduledFor": "12/15/2020, 2:45:00 PM"
+  }
+}
+```
+
+## Create a document from genesis record
+Create a new document, or load a document from it's genesis content (these are equivalent actions). The genesis content may be signed (a DagJWS for the `tile` doctype), or unsigned in some cases.
```

Comment:
> If we had a way to get a CID from some content without actually publishing that content to IPFS

This is possible (technically), don't see how that changes this statement though?

Reply:
Then they wouldn't be equivalent actions anymore.  It would be possible to load a document from its genesis contents without creating the document if it doesn't exist